www/js/diary/infinite_scroll_list.js

```diff
@@ -294,6 +302,12 @@ angular.module('emission.main.diary.infscrolllist',['ui-leaflet',
         $scope.editingTrip = angular.undefined;
     }
 
+    $scope.updateTripProperties = function(trip) {
+      $scope.inferFinalLabels(trip);
+      $scope.updateVerifiability(trip);
+      $scope.recomputeDisplayTrips();
```

Comment:
This is the part that removes the trips from the list once they are labeled correctly, right? I think this should be fine since it doesn't make any remote calls, but we should keep an eye on the performance.

Reply:
`inferFinalLabels` recomputes what inferences we should be showing based on what the user just entered. `updateVerifiability` recomputes whether the verify checkbox should be clickable. `recomputeDisplayTrips` removes trips from `To Label` (or `Unlabeled`, etc.) once they are labeled. There is some inefficiency here as `recomputeDisplayTrips` goes through all trips, whereas we only need to check the trip that was just changed; that would be the optimization if it is necessary.

Reply:
Right, modifying an individual trip is fine, I was just worried about `recomputeDisplayTrips`. But we don't need to optimize that just yet.